rosbag2_transport/include/rosbag2_transport/play_options.hpp

```diff
@@ -30,6 +31,12 @@ struct PlayOptions
   size_t read_ahead_queue_size;
   std::string node_prefix = "";
   float rate = 1.0;
+
+  // Topic names to whitelist when playing a bag. Only messages matching these
+  // specified topics will be played. If list is empty, the filter is ignored
+  // and all messages are played.
```

Comment:
we generally have the rule of having one sentence per line

Reply:
520fd60